challenges/objects-arrays.js

```diff
@@ -7,28 +7,54 @@
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
-
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
-
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
 
+const dinos = [
+  {
+    name: 'tyrannosaurus',
+    diet: 'carnivorous',
+    weight: '7000kg',
+    length: '12m',
+    period: 'Late Cretaceious',
+    speak(){
+      return 'RAWERSRARARWERSARARARRRR!'
+    }
+  },
+  {
+    name: 'stegosaurus',
+    diet: 'herbivorous',
+    weight: '2000kg',
+    length: '9m',
+    period: 'Late Jurassic'
+  },
+  {
+    name: 'velociraptor',
+    diet: 'carnivorous',
+    weight: '15kg',
+    length: '1.8m',
+    period: 'Late Cretaceious'
+  }
+
+]
+
+
 // Using your dinosaur objects, log answers to these questions:
 
 // How much did tyrannosaurus weigh?
-console.log();
+console.log(dinos[0].weight);
 
 // What was the diet of a velociraptor?
-console.log();
+console.log(dinos[2].diet);
 
 // How long was a stegosaurus?
-console.log();
+console.log(dinos[1].length);
 
 // What time period did tyrannosaurus live in?
-console.log();
-
+console.log(dinos[0].period);
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
-console.log();
+console.log(dinos[0].speak());
 
```

Comment:
very, interesting, you didnt need to make your dinos in an array, but it works out so good on you!